src/applications/personalization/profile/components/personal-health-care-contacts/HelpDeskContact.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+const HelpDeskContact = ({ testId }) => (
+  <>
+    <va-telephone
```

Comment:
You may need to also create tests for your new files. I don't see anything testing these.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/radavis/72549-contacts-design-update/src/applications/personalization/profile/components/personal-health-care-contacts/PersonalHealthCareContacts.unit.spec.jsx#L46:L46